src/libraries/System.Security.Cryptography.Encoding/src/System/Security/Cryptography/Base64Transforms.cs

```diff
@@ -23,7 +23,7 @@ public class ToBase64Transform : ICryptoTransform
         // converting to Base64 takes 3 bytes input and generates 4 bytes output
         public int InputBlockSize => 3;
         public int OutputBlockSize => 4;
-        public bool CanTransformMultipleBlocks => false;
+        public bool CanTransformMultipleBlocks => true;
```

Comment:
Remember to submit doc updates: https://docs.microsoft.com/dotnet/api/system.security.cryptography.tobase64transform.cantransformmultipleblocks

(Seriously, I don't know why we doc things like "always returns `false` as an implementation detail." If it's an implementation detail and people shouldn't rely on it, we probably shouldn't doc it.)

Reply:
Thanks.
https://github.com/dotnet/dotnet-api-docs/pull/6855